File: src/priv/bajdcc/LALR1/grammar/runtime/ICodegenByteWriter.java
package priv.bajdcc.LALR1.grammar.runtime;

/**
 * 【中间代码生成】字节码生成
 *
 * @author bajdcc
 */
public interface ICodegenByteWriter {

	public void genInst(RuntimeInst inst);
	public void genOp(int op);
}

File: src/main/kotlin/com/bajdcc/LALR1/grammar/runtime/ICodegenByteWriter.kt
package com.bajdcc.LALR1.grammar.runtime

/**
 * 【中间代码生成】字节码生成
 *
 * @author bajdcc
 */
interface ICodegenByteWriter {

    fun genInst(inst: RuntimeInst)

    fun genOp(op: Int)
}
